prep-html-syntax/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <title>Jar Jar For Supreme Leader</title>
```

Comment:
Be sure to put meta information, such as `<title>` inside of a `<head>` element.

Reply:
Got it and updated!